edocumentation-application/src/test/java/org/dcsa/edocumentation/controller/ShippingInstructionControllerTest.java

```diff
@@ -0,0 +1,93 @@
+package org.dcsa.edocumentation.controller;
+
+import org.dcsa.edocumentation.service.ShippingInstructionService;
+import org.dcsa.edocumentation.transferobjects.*;
+import org.dcsa.edocumentation.transferobjects.enums.*;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.support.TransactionTemplate;
+
+import java.util.List;
+
+// Todo this test should be removed or rewritten to use the controller when implemented when this PR moves beyond draft. Kept here as a convenience for manual testing with the DB.
+@SpringBootTest
+@Disabled
+class ShippingInstructionControllerTest {
+
+  @Autowired ShippingInstructionService shippingInstructionService;
+
+  @Autowired private TransactionTemplate transactionTemplate;
+
+  @Test
+  void test() {
+
+    EquipmentTO equipmentTO = EquipmentTO.builder().equipmentReference("eq_ref_01").build();
+
+    UtilizedTransportEquipmentTO utilizedTransportEquipmentTO = UtilizedTransportEquipmentTO.builder()
```

Comment:
Some of this test data has data factories

Reply:
this test will ultimately be removed when the controller is in place